pkg/image/docker/daemon_provider.go

```diff
@@ -153,30 +154,43 @@ func (p *DaemonImageProvider) pullOptions() (types.ImagePullOptions, error) {
 	}
 	log.Debugf("using docker config=%q", cfg.Filename)
 
-	ref, err := name.ParseReference(p.imageStr)
+	url, err := authURL(p.imageStr)
 	if err != nil {
-		return options, err
+		return options, fmt.Errorf("unable to determine auth hostname: %w", err)
```

Comment:
nit: is hostname the only possible error here?

Reply:
I should change this reference to "url", but no, they can give a bad image reference as well. I'll change this to a warn instead of an error.